impeller/renderer/backend/vulkan/blit_command_vk.cc

```diff
@@ -85,7 +85,20 @@ bool BlitCopyTextureToTextureCommandVK::Encode(
                        image_copy               //
   );
 
-  return true;
+  // If this is not an onscreen, convert back to shader read.
```

Comment:
This comment doesn't match the code (it's the negation of what's written).  It's more correct to say "If it's on screen, don't convert it to shader read".

Reply:
Done